### ISSUE TITLE
The feature extractor EfficientNet training

### ISSUE THREAD

**Issue by LITTLEKKKK** [yfzon/DT-MIL#1]
The EfficientNet is used to get feature embeddings. Are the EfficientNet and the transfomer trained at the same time?
If not, how to train the EfficientNet and what is the training label of EfficientNet?

**Comment by yfzon** [comment]
Please refer to our [MICCAI paper](https://link.springer.com/chapter/10.1007/978-3-030-87237-3_20) for the details. "EfficientNet-B0 pre-trained with ImageNet was employed as the feature extractor and the last few layers of the CNN model were fine-tuned to learn task-relevant features. The transformer part was randomly initiated. The DT-MIL model was trained in an end-to-end manner."
